lib/elixir/lib/agent.ex

```diff
@@ -21,7 +21,7 @@ defmodule Agent do
         end
 
         def value do
-          Agent.get(__MODULE__, & &1)
+          Agent.get(__MODULE__, &(&1))
```

Comment:
Also how do you feel about that. It seems inconsistent that parens aren't always used.

Reply:
nevermind I think this is a pattern!